ui/src/app/views/dataset-page.tsx

```diff
@@ -548,22 +539,29 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
         </FadeBox>
         <FadeBox style={{marginTop: '1rem'}}>
           <div style={{backgroundColor: 'white', border: '1px solid #E5E5E5'}}>
-            <div style={{...styles.selectBoxHeader, display: 'flex', flexDirection: 'row',
-              position: 'relative'}}>
-              <div style={styles.previewDataHeader}>
-                Preview Data Set
+            <div style={{...styles.selectBoxHeader, ...styles.refreshPreviewHeader}}>
+              <div style={{display: 'flex', flexDirection: 'column'}}>
+              <div style={{display: 'flex', alignItems: 'flex-end'}}>
+                <div style={styles.previewDataHeader}>
+                  <div>Preview Data Set</div>
```

Comment:
Do we need div here? can it be <label> or just Preview Data set

Reply:
I can remove the div